<?xml version="1.0"?>
<launch>
    <!-- camera のpose, tfを発行 -->

    <arg name="number" default="1"/>
    <arg name="camera_tf" value="bebop10$(arg number)/camera_optical"/>
    <node pkg="bebop_hatanaka_base" name="virtual_camera" type="virtual_camera.py" output="log"/>
    <node pkg="bebop_hatanaka_base" type="posestamped2tf.py" name="camera_posestamped2tf" output="log" respawn="false" respawn_delay="0">
        <param name="posestamped" value="camera_posestamped"/> 
        <param name="tfname" value="$(arg camera_tf)"/>
    </node>
    <node pkg="bebop_hatanaka_base" name="show_fov_cone" type="show_fov_cone.py" output="log">
      <param name="camera_tf" value="$(arg camera_tf)" />
    </node>
</launch>